src/mono/sample/wasm/browser-bench/main.js

```diff
@@ -203,6 +218,7 @@ try {
 
     const runtime = await dotnet
         .withRuntimeOptions(["--jiterpreter-stats-enabled"])
+        .withInterpreterPgo(true, 30)
```

Comment:
when benchmark is running in frame, we are starting many instances per second. They are not and I think they should not live 30 seconds. Effectively, not testing interpgo? 

In which case it would be better just disable it for the "appstart" scenarios.

Reply:
cc @radekdoulik

Reply:
Interpgo works for successive browser-bench runs locally, you're correct that it doesn't work for appstart but it wouldn't anyway (I tested and nothing in appstart is hot enough to tier). So this is mostly here to allow people to test it locally and measure its impact on a suitably large piece of code.

Reply:
could you please add comment explaining that ?